### DISCUSSION TITLE
Regex quantifiers not working as expected

### DISCUSSION THREAD

**Discussion by ovidiuchirca** [ayoisaiah/f2#89] [Q&A]
Hello,

I came across some problem trying the examples from the tutorial.
Precisely, when I try this:
f2 -f '[ ]{1,}' -r '_'
with one file named "Screenshot from 2022-04-12 14:37:35.png" like in the tutorial,
the quantifier is not working, I get the message "the search criteria didn't match any files".
The problem is with "one or more" quantifier, I have tried {1} for exactly one and it worked perfectly. 
{1,5} doesn't work either

Am I missing something?

Thank you!

**Comment by ayoisaiah** [comment]
@ovidiuchirca Thanks for this! It was due to a bug which has been fixed here: https://github.com/ayoisaiah/f2/pull/88

**Comment by ovidiuchirca** [comment]
Thanks for the reply!
One more thing: is there a way to "trim" the spaces in the filename? I mean
not to remove completely, but to reduce more than one consecutive space to
one space.
I try f2 -f '[ ]{2,}' -r ' ' but it completely removes the spaces...

**Reply by ayoisaiah** [reply]
@ovidiuchirca You are right! The underlying library I'm using was trimming spaces by default. I have fixed that now in [v2.0.3](https://github.com/ayoisaiah/f2/releases). 

Thank you!
